public/modules/graphics/console_mode/console_mode.rb

```diff
@@ -1,87 +1,70 @@
-consoleGraphicalModule  do
-  description {
-    "This is a console mode graphical module for CopyPeste."
-  }
+require 'tty'
+require File.join CopyPeste::Require::Path.base, 'copy_peste/modules.rb'
+require File.join CopyPeste::Require::Path.public, 'tmp_logo'
 
-  author { "Team Framework" }
-
-  usage { 'Just run it!' }
-
-  init {
-    ConsoleMode.new()
-  }
-
-
-  impl {
-    require 'tty'
-
-    %W{ parser console_display }.each do |file|
-      require File.join(
-        CopyPeste::Require::Path.graphics,
-        'console_mode',
-        file
-      )
-    end
+%W{ parser console_display }.each do |file|
+  require File.join(
+    CopyPeste::Require::Path.graphics,
+    'console_mode',
+    file
+  )
+end
 
-    %W{ analysis_requests core_requests }.each do |file|
-      require File.join(
-        CopyPeste::Require::Path.graphics,
-        'console_mode/requests',
-        file
-      )
-    end
+%W{ analysis_requests core_requests }.each do |file|
+  require File.join(
+    CopyPeste::Require::Path.graphics,
+    'console_mode/requests',
+    file
+  )
+end
 
-    # Class accessible through the CopyPeste Core.
-    class ConsoleMode
-      def initialize
-        @alive = true
-        @core_req = CoreRequests.new
-        @analysis_req = AnalysisRequests.new
-      end
+# Class accessible through the CopyPeste Core.
+class ConsoleMode < CopyPeste::Modules::Graphics
+  def initialize
+    super()
+    puts LOGO.blue
 
-      # Set boolean variable used to define if CopyPeste has been run
-      # with or without the debug parameter.
-      def set_debug_mode(debug_mode)
-        ConsoleDisplay.debug = debug_mode
-      end
+    @version = "1.0 - beta"
+    @authors = "CopyPeste Team"
+    @description = "Cli user interface for CopyPeste Framework."
 
-      # Get Hash containing translation between user event's name and CopyPeste
-      # command's names.
-      # @return [Hash] events_to_command hash.
-      def get_events
-        ConsoleDisplay.events_to_command
-      end
+    @core_req = CoreRequests.new
+    @analysis_req = AnalysisRequests.new
+  end
 
-      # This method is called by the core in order to get user events.
-      # When a events is detected, the graphical module format a hash
-      # and return it to the core in order to be executed.
-      # @return [Hash] Hash containing commands informations.
-      def loop
-        cmd = Parser.get_input ConsoleDisplay.prompt
-        cmd_hash = Parser.parse cmd
-        @alive = false if cmd_hash[:cmd] == "exit"
-        cmd_hash
-      end
+  #Set boolean variable used to define if CopyPeste has been run
+  #with or without the debug parameter.
+  def set_debug_mode(debug_mode)
```

Comment:
Setters prototypes are normally created like that: `debug_mode=(debug_mode)` So then in your code you can do debug_mode = true/false